tests/unit/test_pynblint.py

```diff
@@ -1,6 +1,154 @@
 from pynblint.notebook import Notebook
 from pathlib import Path
+import pytest
 
 if __name__ == '__main__':
-    nb = Notebook(Path('../fixtures/my-attempt-at-analytics-vidhya-job-a-thon.ipynb'))
-    print(nb.get_pynblint_results())
+    pytest.main()
+
+
+@pytest.fixture(scope="module")
+def notebooks():
+    nb1 = Notebook(Path("../fixtures") / Path("EmptyNotebook.ipynb"))
```

Comment:
I just read that, in cases like this, you can call the pathlinb.Path constructor only once, e.g.:

```python
Path("../fixtures", "EmtpyNotebook.ipynb")
```

Can you let me know whether this works in Windows too?

Reply:
yes, it works!